tests/libs/estdlib/test_gen_server.erl

```diff
@@ -115,28 +115,21 @@ test_late_reply() ->
         try gen_server:call(Pid, {reply_after, 300, ok}, 200) of
             unexpected -> unexpected
         catch
+            % estdlib
             exit:timeout -> timeout;
-            _A:_B -> unexpected
+            % OTP
+            exit:{timeout, _} -> timeout;
+            T:V -> {unexpected, T, V}
         end,
     %%
     %% flush the late message in the mailbox
+    %% OTP's gen_server(3) man page says it's a two element tuple with a ref as
+    %% a first element.
```

Comment:
That's interesting.  I didn't realize this was even documented.  I assumed the chatter between a client and gen_server was really implementation-defined.